cos/views/views.py

```diff
@@ -33,4 +38,5 @@ def game_view(request):
         player_id = request.session['player_id']
 
     return {'game_id': request.session['game_id'],
-            'player_id': player_id}
+            'player_id': player_id,
+            'game_name': game_name}
```

Comment:
This really should be completed in 2 lines of code. In line 34, it should be:

 `game = request.registry.games.get_game_by_id(request.session['game_id'])`

This is because we already know the game id is valid, otherwise it would have already raised a HTTPNotFound error, which immediately returns.

 Then line 42 would just be: 

`'game_name': game.game_name`

which is just directly grabbing the game name from the game object.